File: sub-web-model/src/main/java/org/github/base/service/impl/TagServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.TagEntity;
import org.github.base.mapper.ITagMapper;
import org.github.base.service.ITagService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-07
 */
@Service
public class TagServiceImpl extends ServiceImpl<ITagMapper, TagEntity> implements ITagService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/TagServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.TagEntity;
import org.github.mysql.blog.base.mapper.ITagMapper;
import org.github.mysql.blog.base.service.ITagService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class TagServiceImpl : ServiceImpl<ITagMapper, TagEntity>(), ITagService
